js/utils.js

```diff
@@ -61,6 +61,14 @@ const showAlert = (message) => {
 
 const isEscapeKey = (evt) => evt.key === 'Escape';
 
+function debounce(callback, timeoutDelay) {
```

Comment:
```suggestion
const debounce = (callback, timeoutDelay) => {
```
у тебя везде стрелочные)